.github/workflows/build.yml

```diff
@@ -403,11 +403,12 @@ jobs:
       - name: Clone
         uses: actions/checkout@v3
 
-      - name: Install Java
+      - name: set up JDK 8
         uses: actions/setup-java@v3
         with:
-          distribution: zulu
           java-version: '8'
+          distribution: 'temurin'
+          cache: gradle
```

Comment:
Why do we have to stick with Java 8? I just checked the error message, and it looks like the minimum Java version required is Java 11 (55.0).
 https://github.com/ggerganov/whisper.cpp/actions/runs/6789530038/job/18456933658?pr=1382#step:4:9